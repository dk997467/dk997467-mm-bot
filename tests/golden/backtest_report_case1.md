| key | value |
|---|---|
| fills_total | 182.000000 |
| net_bps | 364.000000 |
| taker_share_pct | 0.000000 |
| order_age_p95_ms | 0.000000 |
| fees_bps | -364.000000 |
| turnover_usd | 9145.500000 |
