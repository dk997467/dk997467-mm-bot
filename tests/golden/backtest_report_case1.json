{"fees_bps":-364.0,"fills_total":182,"net_bps":364.0,"order_age_p95_ms":0.0,"runtime":{"mode":"backtest","utc":"1970-01-01T00:00:00Z","version":"0.1.0"},"taker_share_pct":0.0,"turnover_usd":9145.5}
